.github/actions/setup-job-docker/action.yml

```diff
@@ -0,0 +1,39 @@
+name: 'setup job'
+description: 'Setup dart and/or rust CI job'
+inputs:
+  dart:
+    description: 'Setup dart'
+    default: "false"
+    required: false
+  flutter:
+    description: 'Setup flutter'
+    default: "false"
+    required: false
```

Comment:
⛏️ 
we could also have just one of them and name it `dart-deps` or keep the name `dart`? wdyt?